src/workspaces/dashboard/WorkspaceDashboard.ts

```diff
@@ -43,77 +43,87 @@ export const WorkspaceDashboard = forwardRef(
     // @ts-expect-error
     const { value: canEdit } = canEditWorkspace(workspace);
 
-    return div({ style: { gridTemplateColumns: 'auto min-content', gridTemplateRows: '1fr 2fr', display: 'grid' } }, [
```

Comment:
I was trying to ensure that the rectangle BELOW the RHS dashboard components extended far enough (as otherwise the background color was wrong). However, by letting those 2 rows grow so much I ended up with extra space.

This was fixed by giving extra space to only the bottom (empty) one, while also setting minHeight to 100%. And then I removed padding on the (blank) bottom row so that it didn't artificially increase the size of that row.